src/clr-angular/forms/datepicker/datepicker.module.ts

```diff
@@ -27,7 +28,10 @@ export const CLR_DATEPICKER_DIRECTIVES: Type<any>[] = [
 ];
 
 @NgModule({
-    imports: [CommonModule, ClrHostWrappingModule, ClrConditionalModule, ClrIconModule, ClrFocusTrapModule],
+    imports: [
```

Comment:
@youdz @gnomeontherun: Marking this so that I can make sure you review this.

Reply:
@youdz don't we want to import the `ClrCommonFormsModule` here instead of `ClrHostWrappingModule`?

Reply:
We're not using it, are we? And I don't thing `ClrHostWrappingModule` imports the `ClrCommonFormsModule` either. I'd rather import stuff only when we need to import it, and export extra modules when they're obvious dependencies. So I think what @adityarb88 has now is fine.